profiler/lib.profiler/src/org/netbeans/lib/profiler/heap/ClassDumpSegment.java

```diff
@@ -202,12 +202,12 @@ void addInstanceSize(ClassDump cls, int tag, long instanceOffset) {
         }
     }
 
-    synchronized List /*<JavaClass>*/ createClassCollection() {
+    synchronized List/*<JavaClass>*/ createClassCollection() {
```

Comment:
This is package private - I would not consider this API, so could be made generic and then the callsite of this function could be generified.

Reply:
> This is package private - I would not consider this API, so could be made generic and then the callsite of this function could be generified.

Fixed..